src/server/seeds/development/02_products.js

```diff
@@ -45,6 +45,7 @@ exports.seed = function (knex) {
           price: 80,
           color: 'White',
           size: 'Extra-Large',
+          status: 'running_low',
```

Comment:
for this id , **status** should be **out of stock** according to issue #147

Reply:
When you have stock.. you cant keep status out_of_stock.. Thats why i changed it

Reply:
read the issue plz.